prometheus/prometheus.rules.yml

```diff
@@ -152,6 +152,16 @@ groups:
     annotations:
       description: 'Some operation failed due to consistency level'
       summary: Consistency Level error
+  - alert: preparedCacheEviction
+    expr: sum(rate(scylla_cql_prepared_cache_evictions[2m])) by (cluster) + sum(rate(scylla_cql_authorized_prepared_statements_cache_evictions[2m])) by (cluster) > 3
+    for: 5m
+    labels:
+      severity: "1"
+      advisor: "preparedEviction"
+      dashboard: "scylla-detailed"
+    annotations:
+      description: 'The prepared-statement cache is being continuously evicted, which could indicate a problem in your prepared-statement.'
```

Comment:
All looks good.
A tiny nitpick: in the sentence above I'd add the words "usage logic" at the end:
"The prepared-statement cache is being continuously evicted, which could indicate a problem in your prepared-statement usage logic."

Reply:
I marked as "request a change" but if you disagree with my comment about I don't want to argue and I don't mind if merge this PR as-is. ;)